hardhat.config.ts

```diff
@@ -66,7 +66,7 @@ const config: HardhatUserConfig = {
     },
     eth: {
       url: `https://ultra-empty-sanctuary.quiknode.pro/${QUICKNODE_ETH_MAINNET_API_KEY}`,
-      accounts: [`${MAINNET_PRIVATE_KEY_FOR_CONTRACT_DEPLOYMENT}`]
+      accounts: [`${PRIVATE_KEY_FOR_V4_CONTRACT_DEPLOYMENT}`]
```

Comment:
I think we also need to check if https://ultra-empty-sanctuary.quiknode.pro/${QUICKNODE_ETH_MAINNET_API_KEY} is configured properly in the github secrets. Otherwise we can use a public url

Reply:
Seems like the url should be there as it was added in Oct 2023